models/stg_sap.yml

```diff
@@ -0,0 +1,2158 @@
+      - name: ind02
+        description: Indicator for indirect valuation
+      - name: ind03
+        description: Indicator for indirect valuation
+      - name: ind04
+        description: Indicator for indirect valuation
+      - name: ind05
+        description: Indicator for indirect valuation
+      - name: ind06
+        description: Indicator for indirect valuation
+      - name: ind07
+        description: Indicator for indirect valuation
+      - name: ind08
+        description: Indicator for indirect valuation
+      - name: ind09
+        description: Indicator for indirect valuation
+      - name: ind10
+        description: Indicator for indirect valuation
+      - name: ind11
+        description: Indicator for indirect valuation
+      - name: ind12
+        description: Indicator for indirect valuation
+      - name: ind13
+        description: Indicator for indirect valuation
+      - name: ind14
+        description: Indicator for indirect valuation
+      - name: ind15
+        description: Indicator for indirect valuation
+      - name: ind16
+        description: Indicator for indirect valuation
+      - name: ind17
+        description: Indicator for indirect valuation
+      - name: ind18
+        description: Indicator for indirect valuation
+      - name: ind19
+        description: Indicator for indirect valuation
+      - name: ind20
+        description: Indicator for indirect valuation
+      - name: ind21
+        description: Indicator for indirect valuation
+      - name: ind22
+        description: Indicator for indirect valuation
+      - name: ind23
+        description: Indicator for indirect valuation
+      - name: ind24
+        description: Indicator for indirect valuation
+      - name: ind25
+        description: Indicator for indirect valuation
+      - name: ind26
+        description: Indicator for indirect valuation
+      - name: ind27
+        description: Indicator for indirect valuation
+      - name: ind28
+        description: Indicator for indirect valuation
+      - name: ind29
+        description: Indicator for indirect valuation
+      - name: ind30
+        description: Indicator for indirect valuation
+      - name: ind31
+        description: Indicator for indirect valuation
+      - name: ind32
+        description: Indicator for indirect valuation
+      - name: ind33
+        description: Indicator for indirect valuation
+      - name: ind34
+        description: Indicator for indirect valuation
+      - name: ind35
+        description: Indicator for indirect valuation
+      - name: ind36
+        description: Indicator for indirect valuation
+      - name: ind37
+        description: Indicator for indirect valuation
+      - name: ind38
+        description: Indicator for indirect valuation
+      - name: ind39
+        description: Indicator for indirect valuation
+      - name: ind40
+        description: Indicator for indirect valuation
```

Comment:
Are these all intended to be the same description?

Reply:
Yep: 
<img width="677" alt="Screenshot 2023-07-06 at 4 25 53 AM" src="https://github.com/fivetran/dbt_sap_source/assets/108772760/b2183b89-0bb1-4b1c-be52-ece37369f312">